golem-rib/src/parser/pattern_match.rs

```diff
@@ -18,6 +18,7 @@ use combine::{attempt, not_followed_by, sep_end_by1, ParseError, Parser};
 use match_arm::*;
 
 use crate::expr::Expr;
+use crate::parser::comment::comments;
```

Comment:
for pattern matches, not every token is recursively going back to rib_expr and hence we support comments there explicitly.